src/main/java/com/rultor/profiles/Profiles.java

```diff
@@ -83,28 +81,23 @@ public Profile fetch(final Talk talk) throws IOException {
      * Fetch a profile from an XML.
      * @param xml The XML
      * @return Profile found
+     * @todo #1039:30min Read .rultor.yml for authentication purposes from
```

Comment:
@original-brownbear thanks, so we share the same concern
@xupyprmv sorry, but we can't use PDD like that. After each PR & puzzle application should be production ready.